nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss fix navbar 31

# Description

## Type of change


# Testing Instructions

Fetch the repo and checkout into my branch ss-fix-navbar-31 and reduce your screen size to 360p (in the width section at the top of page in dev tools) and see if it is responsive.

# Checklist:
